sql/core/src/test/scala/org/apache/spark/sql/sources/InsertSuite.scala

```diff
@@ -441,7 +441,6 @@ class InsertSuite extends DataSourceTest with SharedSQLContext {
         sql(
           s"""
              |INSERT OVERWRITE LOCAL DIRECTORY '$path'
-             |STORED AS orc
```

Comment:
Why did you delete this?

Reply:
if we use `stored as` and file format is orc or parquet it will be converted to data source flow.

Reply:
I believe the test case name should be modified correspondingly and a new test case for (orc/parquet) should be added.

Reply:
@advancedxy i have updated the testcase name and for orc and parquet testcase was already added. As they will be converted to data source,  the data in the directory would be compressed.